.github/workflows/benchmark.yml

```diff
@@ -17,13 +17,14 @@ jobs:
   benchmark:
     runs-on: ubuntu-latest
     env:
+      GH_EVENT: ${{github.event_name}}
       GH_REF: ${{github.head_ref}}
       GH_REPO: ${{github.repository}}
-      GH_EVENT: ${{github.event_name}}
+      EXECUTION_ENV: github-action
     strategy:
       fail-fast: false
       matrix:
-        ruby: [jruby-9.2, 2.6, 2.7, 3.2, 3.3]
+        ruby: [jruby-9.4, 3.3]
```

Comment:
Probably a bigger discussion on what versions we want to track against.  I see a point in not necessarily tracking this data for every ruby version, esp on PRs.  But for continuity over time, we likely want to keep a complete list for releases (ie, maybe we need separate tasks that run on version-3 branch vs PRs?).  But we can address that in the future.